msteams-platform/sbs-botbuilder-linkunfurling.yml

```diff
@@ -276,14 +276,14 @@ items:
 
        :::image type="content" source="../msteams-platform/assets/images/Teams-link-unfurling/authorizedscope.png" alt-text="Add client application":::
       
-       The following image displays the **Client Id**:
+       The following image displays the **Client ID**:
 
        :::image type="content" source="../msteams-platform/assets/images/meetings-side-panel/clientapps.png" alt-text="Client app"::: 
          
     1. In the left panel, select **API Permissions**. 
 
        > [!NOTE]
-       > Users need to consent to the API permissions only if the AAD app is registered in a different tenant.
```

Comment:
Good catch here, to fix this branding issue! Thank you.

I thought we fixed all instances of branding issue - why is this AAD instance here. Was it missed in the first wave of fixes or was it added later?

Reply:
It was missed I guess Ashish.